xline/src/rpc/mod.rs

```diff
@@ -71,6 +71,7 @@ pub(crate) use self::etcdserverpb::{
     compare::{CompareResult, CompareTarget, TargetUnion},
     kv_server::{Kv, KvServer},
     lease_server::{Lease, LeaseServer},
+    range_request::{SortOrder, SortTarget},
```

Comment:
only include range_request. And use range_request::SortOrder and range_request::SortTarget.